polypomo

```diff
@@ -133,6 +138,57 @@ class Status:
             self.timer = Timer(self.worktime)
 
 
+@contextmanager
+def setup_conn():
+    global DATABASE_PATH
+    # check if the database exist
+    if (os.path.exists(DATABASE_PATH)):
+        print('database already exist')
```

Comment:
Please consider removing these and other print statements below that read like debugging output